web/packages/teleport/src/features.tsx

```diff
@@ -810,7 +841,14 @@ export class FeatureHelpAndSupport implements TeleportFeature {
     getLink() {
       return cfg.routes.support;
     },
-    searchableTags: ['help', 'support', NavTitle.HelpAndSupport],
```

Comment:
should we keep `NavTitle.HelpAndSupport` in here? i guess not if we already have "help" and "support"

Reply:
I removed this from here because with the other change in this PR, we're going to start search indexing the nav title for all features automatically, so there's no need to include it in the `searchableTags`. The functionality is remaining the same

https://github.com/gravitational/teleport/blob/7526e8b62906350baaa1ad8831f02b477bb80eff/web/packages/teleport/src/Navigation/Search.tsx#L136-L138